src/main/java/com/wolApi/wolApi/api/bibleVerses/BibleVerseViews.java

```diff
@@ -21,20 +21,23 @@ public class BibleVerseViews {
     public BibleVerseViews(BibleVerseService bibleVerseService) {
         this.bibleVerseService = bibleVerseService;
     }
-    @GetMapping("/get-verse/{book}/{chapter}/{verse}")
-    public Verse getVerse(@PathVariable("book") String book,
-                          @PathVariable("chapter") String chapter,
-                          @PathVariable("verse") String verse) throws IOException {
+    @GetMapping("/get-verse/{bookNum}/{chapterNum}/{verseNum}")
```

Comment:
These have trouble when you add a trailing `/` at the end. They have to follow the path format specified. Instead, I propose using query parameters as they are more sustainable.

Reply:
Look at my below comments for the query bits

Reply:
Yeah it would make more sense to use query params